jcommune-view/jcommune-web-view/src/main/webapp/resources/javascript/app/plainToLink.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2011  JTalks.org Team
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+/**
+ * Link to website in message content displayed as hyperlink automatically
+ */
+function plainToLink() {
+    $('.post-content-body').each(function(indx, element){
+        element.innerHTML = element.innerHTML.replace(/(\b(https?):\/\/[-A-Z0-9+&@#\/%?=~_|!:,.;]*[-A-Z0-9+&@#\/%=~_|])/gi,
```

Comment:
I've pushed improved version with enhanced URL recognizion and some bugfixes. In previouse version links which already inside <a> and <img> tags were incorrectly replaced.